ReactWindows/ReactNative/Tracing/Tracer.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using System;
+using Newtonsoft.Json;
```

Comment:
Also,remove Newtonsoft.Json.

---
In reply to: [71040179](https://github.com/ReactWindows/react-native-windows/pull/530#discussion_r71040179) [](ancestors = 71040179)